### PR TITLE
Port awapi awiv adapter to .auto

### PR DESCRIPTION
**Pull request by isamu-takagi**
## Remarks

これのマージが必要。https://github.com/tier4/autoware_iv_msgs/pull/58

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [pull request guidelines][pull-request-guidelines]
- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by kenji-miyake** [comment]
パッと見良さそうな感じがしました。
msgsがマージされてビルドが通るのを待ちたいと思います。

**Comment by tkimura4** [comment]
@isamu-takagi 
rebaseかmergeか何かでこちらの取り込みお願いします
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/558

File: awapi/awapi_awiv_adapter/src/awapi_awiv_adapter_core.cpp [hunk 1]
```diff
@@ -62,30 +62,37 @@ AutowareIvAdapter::AutowareIvAdapter()
...
+      "input/vehicle_cmd", durable_qos,
+      std::bind(&AutowareIvAdapter::callbackVehicleCmd, this, _1));
+  sub_turn_indicators_ =
+    this->create_subscription<autoware_auto_vehicle_msgs::msg::TurnIndicatorsReport>(
+      "input/turn_indicators", 1, std::bind(&AutowareIvAdapter::callbackTurnIndicators, this, _1));
+  sub_hazard_lights_ =
+    this->create_subscription<autoware_auto_vehicle_msgs::msg::HazardLightsReport>(
+      "input/hazard_lights", 1, std::bind(&AutowareIvAdapter::callbackHazardLights, this, _1));
   sub_twist_ = this->create_subscription<geometry_msgs::msg::TwistStamped>(
     "input/twist", 1, std::bind(&AutowareIvAdapter::callbackTwist, this, _1));
```
Comment (tkimura4) [inline awapi/awapi_awiv_adapter/src/awapi_awiv_adapter_core.cpp:78]: twist -> odom ...??

Comment (isamu-takagi) [reply awapi/awapi_awiv_adapter/src/awapi_awiv_adapter_core.cpp]: https://github.com/tier4/AutowareArchitectureProposal.iv/pull/554/commits/cecb320f149a84029f068873bee0447f50f6d405 オドメトリに変更しました。

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**